packages/ui/src/components/molecules/SummaryCard.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { FC } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import { EditButton } from './Prefabs';
+
+import {
+  ClockIcon,
+  EmailIcon,
+  EncryptedMessageIcon,
+  MessageIcon,
+  SvgProps,
+  UserIcon,
+  WalletIcon,
+} from '../../assets';
+import { Flex, NomineeMessage, Divider, Typography } from '../atoms';
+import { svgGradients } from '../GlobalStyles';
+
+const iconMap: Record<string, React.FC<SvgProps>> = {
+  wallet: WalletIcon,
+  clock: ClockIcon,
+  user: UserIcon,
+  email: EmailIcon,
+  message: MessageIcon,
+  encrypted: EncryptedMessageIcon,
+};
+
+export type IconType = keyof typeof iconMap;
+export type BackgroundType = 'gold' | 'silver' | 'default';
+
+export interface SummaryField {
```

Comment:
extend UtilsProps such that the user can change width, height, spacing and other properties.

```suggestion
export interface SummaryField extends UtilsProps {
```

Reply:
updated